vpn_slice/mac.py

```diff
@@ -102,8 +102,8 @@ def set_link_info(self, device, state, mtu=None):
         self._ifconfig(*args)
 
     def add_address(self, device, address):
+        address = ip_interface(address)
         if address.version == 6:
-            family = 'inet6'
+            self._ifconfig(device, 'inet6', address)
         else:
-            family = 'inet'
-        self._ifconfig(device, family, ip_network(address), address)
+            self._ifconfig(device, 'inet', address.ip, address.ip, 'netmask', '255.255.255.255')
```

Comment:
Looks good. I did a double-take at the repetition of the IP address, but it appears from https://gitlab.com/openconnect/vpnc-scripts/blob/master/vpnc-script#L193 that this is indeed the correct syntax. :+1: